<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>601</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>lineID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>stationID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>userID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>deviceID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>payType</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>101</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>lineID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>stationID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string>userID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="text">
        <string>status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="text">
        <string>deviceID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string>payType</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>20</y>
      <width>251</width>
      <height>290</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>end time</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateTimeEdit" name="endTime"/>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="startTime"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>start time</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Time Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QCheckBox" name="type_3">
         <property name="text">
          <string>Type 2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="type_1">
         <property name="text">
          <string>Type 0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="type_4">
         <property name="text">
          <string>Type 3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="type_2">
         <property name="text">
          <string>Type 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Pay Type Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Line Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lineA">
         <property name="text">
          <string>Line A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lineB">
         <property name="text">
          <string>Line B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lineC">
         <property name="text">
          <string>Line C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="set_filter">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="set_clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
